### DISCUSSION TITLE
Collaboration and shared lists?

### DISCUSSION THREAD

**Discussion by TruDataRx-Gary** [karakeep-app/karakeep#518] [General]
Are there any plans to include a way to collaborate and share lists in hoarder?

**Comment by Sulcalibur** [comment]
This is what I came to post. Loving Hoarder and a public shared list would be amazing.

**Comment by MohamedBassem** [comment]
They are both on our roadmap. You can track the progress here:

#37 for collab lists, and #126 for public lists.
